### ISSUE TITLE
OTA update

### ISSUE THREAD

**Issue by jsponz** [martin-ger/esp_wifi_repeater#324]
Is it possible that the esp checks regularly for a new update and if found makes the update process automatically?

Is there any way to force the update? i.e. Sending a command via MQTT.

**Comment by martin-ger** [comment]
No, there is no auto-update funktionality. However you can initiate an over-the-air-update (https://github.com/martin-ger/esp_wifi_repeater#ota-over-the-air-update-support ).

You can trigger such an update (and also any other command) via MQTT by sending a string "ota update" to a configured command-topic. https://github.com/martin-ger/esp_wifi_repeater#mqtt-support
